### ISSUE TITLE
custom dialog / group id on creation ? 

### ISSUE THREAD

**Issue by richanshah** [ConnectyCube/connectycube-flutter-samples#298]
@TatankaConCube, while creating a chat dialog ( for type group) can I pass custom group/dialog ID as per my requirement? , I have tried but it is generating a unique id only not the one i am passing.

**Comment by richanshah** [comment]
CubeDialog newDialog = CubeDialog(
            dialogId: dialogId
            CubeDialogType.GROUP,
            name: groupName ?? name,
            photo: groupPhoto ?? "",
            occupantsIds: usersList,
          )
          
          there is a feild for that but it is not taking it , it is generating new id only.

**Comment by richanshah** [comment]
POST https://api.connectycube.com/chat/Dialog 
15:01:56.662                                      I  HEADERS
15:01:56.662                                      I    {Content-type: application/json, ConnectyCube-REST-API-Version: 0.1.1, CB-SDK: Flutter 2.7.0, CB-Token: 325027CF9A6EB6F6FE33DD6CFB5DC9AF3280}
15:01:56.663                                      I  BODY
15:01:56.663                                      I    {"photo":"","name":"vittals","type":2,"occupants_ids":[10551524],"data":{"groupUuid":"a4624005-6b47-4327-86e4-b66a9acb7a4d","class_name":"CustomDataClassName"}}
15:01:56.663                                      I  
15:01:57.398                                      I  CB-SDK: : *********************************************************
15:01:57.398                                      I  *** RESPONSE *** 404 *** a91fb10a-4d71-45a0-b736-2c80225b5788 ***
15:01:57.398                                      I  HEADERS
15:01:57.399                                      I    {connection: keep-alive, content-type: application/json; charset=utf-8, date: Wed, 06 Sep 2023 09:31:59 GMT, strict-transport-security: max-age=15768000; includeSubDomains, server: nginx/1.25.0, content-length: 30}
15:01:57.399                                      I  BODY
15:01:57.399                                      I    {"errors":["Class not found"]}

**Comment by richanshah** [comment]
why i am getting class not found error

**Comment by richanshah** [comment]
CubeDialogCustomData customData =
          CubeDialogCustomData("CustomDataClassName"); 
          
          it is same as documents , can u provide me example ?

**Comment by TatankaConCube** [comment]
please follow our [documentation](https://developers.connectycube.com/server/chat?id=custom-parameters) on how to add the custom data to the `CubeDialog` model

**Comment by richanshah** [comment]
@TatankaConCube  , I dont see option in admin panel for creating this? do  i have to create there first?

**Comment by richanshah** [comment]
"data": {
    "class_name": "CUSTOM_DIALOG_PARAMS",
    "dialog_number": 101,
    "optional_name": "My first dialog with friends"
  }
  
  do i  have to create class like this and pass it to this ? 
 CubeDialogCustomData customData =
CubeDialogCustomData("CUSTOM_DIALOG_PARAMS");

**Comment by TatankaConCube** [comment]
yes, you should create your class scheme first in the admin panel, then you can create the instance of `CubeDialogCustomData` using this scheme

**Comment by TatankaConCube** [comment]
> I dont see option in admin panel for creating

go to the Admin panel -> Custom -> List and press the "Add" button

**Comment by richanshah** [comment]
ya okay thanks

**Comment by richanshah** [comment]
> Admin panel -> Custom -> List

I think we need to upgrade plan for this @TatankaConCube
